src/modules/mc_att_control/mc_att_control.hpp

```diff
@@ -58,6 +58,8 @@
 #include <uORB/topics/vehicle_land_detected.h>
 #include <uORB/topics/landing_gear.h>
 
+#include <AttitudeControl.hpp>
```

Comment:
Within a module, I somewhat prefer relative includes, which makes it easier to see where the file is and reduces potential for conflicts:
```suggestion
#include "AttitudeControl/AttitudeControl.hpp"
```

Reply:
Thanks for the quick reviewing! I totally see your point if the class is closely tied to the module and lives within it's CMake target. But the whole point of this pr is to modularize the controller into a reusable self-contained library (https://github.com/PX4/Firmware/pull/11308/files#diff-66b6bc5cf0e74d3ebb07ebb30d50ade2R34) that is included and linked by CMake via the separate `AttitudeControl` library target (https://github.com/PX4/Firmware/pull/11308/files#diff-59b5eaaeffc3071a322da5225c7e3706R48). This makes it file location independent as long as the subdirectory is recursively added (https://github.com/PX4/Firmware/pull/11308/files#diff-59b5eaaeffc3071a322da5225c7e3706R34). If the file location is a problem we can already move it to the libraries directory but I thought I leave it in the mc_att_control folder for now since this is the first step only refactoring the existing functionality. I wanted to separate this step from introduction of testing, extending and reusing to make overseeable prs which are easy to review. (Testing draft here: https://github.com/PX4/Firmware/pull/11573)